pubtools/pulplib/_impl/model/repository/base.py

```diff
@@ -479,6 +479,11 @@ def remove_content(self, **kwargs):
 
                 If omitted, content of all types will be removed.
 
+            filters (:class:`~pubtools.pulplib.Criteria`)
```

Comment:
This should be called `criteria` and not `filters` - that's what it's called for all the similar methods like search_content, copy_content.

Also could you please document this as the first argument?  I explained a bit more in another comment, but summarized - the idea was that, once this function supports a Criteria object, then that's the preferred way of using it.  type_ids might eventually be deprecated and removed since we don't really need both.  As the most important/preferred way of using the method now, it'd be best to mention the criteria arg first.

Reply:
Done.